mixins/test/localize-mixin.html

```diff
@@ -61,74 +112,41 @@
 					});
 				});
 
-				describe('localize', () => {
```

Comment:
I kept all these tests, but just put them inside the `forEach` that runs them for both static and async versions.